LICENCE-THIRD-PARTY

```diff
@@ -32,6 +59,15 @@ licensed under MIT licence.
 third-party-licences/Detectorjs.txt
 ==============================================================================
 
+==============================================================================
+Mathjs
+------------------------------------------------------------------------------
+https://github.com/josdejong/mathjs
+Copyright 2013-2019 Jos de Jong 
+Licensed under the Apache License, Version 2.0
```

Comment:
'licenced' is lowercase everywhere else - I can't remember why I made it that way, did CSU do that?

Reply:
Sorry, 'licensed' - as its a verb